docs/src/DeveloperDocumentation/printing_details.md

```diff
@@ -130,7 +130,7 @@ function Base.show(io::IO, R::NewRing)
 end
 ```
 And this is how it looks like:
-```julia
+```julia-repl
```

Comment:
why is this not a doctest? together with the code blocks right above, this could be made to work

Reply:
I am sure it could be made to work, but I already put more work into this trivial quick cleanup of incorrectly rendered documentation than I care for, and untangling this and iterating on this until it works does not seem like the best use of my time.